aptos-move/framework/aptos-names/sources/config.move

```diff
@@ -0,0 +1,434 @@
+*/
+
+module aptos_names::config {
+    friend aptos_names::domains;
+
+    use aptos_framework::account;
+    use aptos_framework::aptos_account;
+    use aptos_names::utf8_utils;
+    use aptos_token::property_map::{Self, PropertyMap, PropertyValue};
+    use std::error;
+    use std::signer;
+    use std::string::{Self, String};
+
+    const CONFIG_KEY_ENABLED: vector<u8> = b"enabled";
+    const CONFIG_KEY_ADMIN_MULTISIG_ADDRESS: vector<u8> = b"admin_multisig_address";
+    const CONFIG_KEY_FOUNDATION_FUND_ADDRESS: vector<u8> = b"foundation_fund_address";
+    const CONFIG_KEY_TYPE: vector<u8> = b"type";
+    const CONFIG_KEY_CREATION_TIME_SEC: vector<u8> = b"creation_time_sec";
+    const CONFIG_KEY_EXPIRATION_TIME_SEC: vector<u8> = b"expiration_time_sec";
+    const CONFIG_KEY_MAX_NUMBER_OF_YEARS_REGISTERED: vector<u8> = b"max_number_of_years_registered";
+    const CONFIG_KEY_MAX_DOMAIN_LENGTH: vector<u8> = b"max_domain_length";
+    const CONFIG_KEY_TOKENDATA_DESCRIPTION: vector<u8> = b"tokendata_description";
+    const CONFIG_KEY_TOKENDATA_URL_PREFIX: vector<u8> = b"tokendata_url_prefix";
+    const CONFIG_KEY_DOMAIN_PRICE_PREFIX: vector<u8> = b"domain_price_";
+
+    const DOMAIN_TYPE: vector<u8> = b"domain";
+    const SUBDOMAIN_TYPE: vector<u8> = b"subdomain";
+
+    const COLLECTION_NAME_V1: vector<u8> = b"Aptos Names V1";
+
+    /// Raised if the signer is not authorized to perform an action
+    const ENOT_AUTHORIZED: u64 = 1;
+    /// Raised if there is an invalid value for a configuration
+    const EINVALID_VALUE: u64 = 2;
+
+    struct ConfigurationV1 has key, store {
+        config: PropertyMap,
+    }
+
+    public(friend) fun initialize_v1(framework: &signer) acquires ConfigurationV1 {
+        move_to(framework, ConfigurationV1 {
+            config: property_map::empty(),
+        });
+
+        // TODO: SET THIS TO SOMETHING REAL
+        // We set it directly here to allow boostrapping the other values
+        set_v1(@aptos_names, config_key_admin_multisig_address(), &@aptos_names);
+        set_v1(@aptos_names, config_key_foundation_fund_address(), &@aptos_names);
+
+        set_is_enabled(framework, true);
+
+        set_max_number_of_years_registered(framework, 10u8);
+        set_max_domain_length(framework, 63);
+
+        // TODO: SET THIS TO SOMETHING REAL
+        set_tokendata_description(framework, string::utf8(b"This is an official Aptos Foundation Name Service Name"));
+        set_tokendata_url_prefix(framework, string::utf8(b"https://aptosnames.com/api/v1/metadata/"));
+
+        // TODO: SET REAL VALUES FOR DOMAIN PRICES
+        set_domain_price_for_length(framework, (100 * octas()), 2);
+        set_domain_price_for_length(framework, (60 * octas()), 3);
+        set_domain_price_for_length(framework, (30 * octas()), 4);
+        set_domain_price_for_length(framework, (15 * octas()), 5);
+        set_domain_price_for_length(framework, (5 * octas()), 6);
+
+        // These are for testnet only
```

Comment:
Why not just get rid of these and then separately manually update these in testnet?